src/utils.rs

```diff
@@ -1,30 +1,14 @@
-use anyhow::anyhow;
```

Comment:
We can probably remove much of the `utils` because they are redundant with the `fvm_utils::runtime`